packages/web-lib/src/persistence/strategies/local-storage.strategy.ts

```diff
@@ -0,0 +1,69 @@
+import { IStorageStrategy, OperationResult } from '../web-store.types';
+
+export class LocalStorageStrategy implements IStorageStrategy {
+  storage: Storage;
+
+  constructor(storage?: Storage) {
+    this.storage = storage ?? localStorage;
+  }
+
+  async init(): Promise<void> {}
+
+  async getOneByKey<TData>(key: string): Promise<OperationResult<TData>> {
+    try {
+      const storedItem = this.storage.getItem(key);
+      if (!storedItem) {
+        return { success: false, data: 'nothing' as TData };
+      }
+      return {
+        success: true,
+        data: JSON.parse(storedItem) as TData,
```

Comment:
good start, we will additioanally add custom reviwers for non-serializable values, such as Map, Set, Date etc